Dockerfile

```diff
@@ -18,13 +19,16 @@ RUN curl -o /usr/local/bin/gomplate -sSL https://github.com/hairyhenderson/gompl
 RUN chmod 755 /usr/local/bin/gomplate
 
 RUN mkdir /dependencies
-RUN wget -P /dependencies https://laotzu.ftp.acc.umu.se/cdimage/archive/12.0.0/amd64/iso-cd/debian-12.0.0-amd64-netinst.iso
+RUN wget -P /dependencies ${ISO_DOWNLOAD_LINK}
 COPY dependencies/vmware-gosc_12.1.0.25580-20029049_amd64.deb /dependencies
 
 ARG DEBIAN_FRONTEND=noninteractive
 RUN apt install -y pipx
 
-RUN useradd -m ray
+#RUN useradd -m ray
+RUN adduser --disabled-password --gecos '' ray
+RUN adduser ray sudo
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
What will be the issue if we don't do this?

Reply:
It is more convenience to have this for debug purpose.